src/qml/imports/Theme/QfMenu.qml

```diff
@@ -0,0 +1,23 @@
+import QtQuick
+import QtQuick.Controls
+
+Menu {
+  id: control
+
+  property int paddingMultiplier: 1
```

Comment:
We could add a property bool skipFirstRow to allow for us to use QfMenu for the main menu / map canvas menu / etc.

Then we'd need a property int minimumRowWidth to pass on a customized first row's width.